src/System.Text.Json/tests/JsonTestHelper.cs

```diff
@@ -6,12 +6,14 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
+using System.Text.Json.Tests;
+using System.Text.RegularExpressions;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Xunit;
 using Xunit.Sdk;
 
-namespace System.Text.Json.Tests
+namespace System.Text.Json
```

Comment:
Why was this change necessary?

Reply:
Because I'm lazy. :P I wanted to get the helpers into the root namespace without having to rename all of the other test files which (for some reason) put the `Tests` name at the end of the namespace.

Reply:
Oh, I see. `System.Text.Json.Tests` vs `System.Text.Json.Serialization.Tests`.